force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +308,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
-
+        Integer totalPositiveNums = 0;
         // Loop through the list of integers
             // if the number is negative skip this iteration
-        
-
-        return null; // Replace null with the variable you used to store the result
+        for(Integer i = 0; i < numbers.size(); i++) {
+            If (numbers[i] > 0) {
+                totalPositiveNums += numbers[i];
+            }
+        }
+        return totalPositiveNums; 
```

Comment:
You can iterate over the List itself which saves you from managing the `i` loop variable. You'll need to decide which method is best on a case-by-case basis. In this specific case, I would recommend iterating over the list itself